source/manual/fix-out-of-date-search-indices.html.md

```diff
@@ -15,21 +15,22 @@ and `unpublish` messages that have not been processed need to be resent.
 Content in the `govuk` index is populated from the [Publishing API message queue][queue].
 Missing documents can be recovered by resending the content to the message queue. In the
 Publishing API, run the following rake task (including the quotes) to replay traffic between
-two datestamps:
+two datestamps (after SSH'ing into the `publishing_api` machine):
 
 ```
+cd /var/apps/publishing-api && \
 govuk_setenv publishing-api bundle exec \
 rake 'represent_downstream:published_between[2018-12-17T01:02:30, 2018-12-18T10:20:30]'
 ```
 
-You can also run this task from Jenkins.
+You can also [run this task from Jenkins](https://deploy.blue.production.govuk.digital/job/run-rake-task/parambuild/?TARGET_APPLICATION=publishing-api&MACHINE_CLASS=publishing_api&RAKE_TASK=represent_downstream:published_between[2018-12-17T01:02:30,%202018-12-18T10:20:30]).
 
```

Comment:
We could add a link to each environment which may be helpful: https://github.com/alphagov/govuk-developer-docs/blame/322891a0379b180e10e2cb6593e502a76bf729bd/source/manual/howto-change-slug-and-create-redirect.html.md#L21-L28

Reply:
We _could_, but we do this very inconsistently throughout the docs. It also requires changing the file type (to end in `.erb`) which is a pain for version control diffs. I think I'll just leave this one 👍

Reply:
The example I linked above just uses Markdown, rather than a Ruby helper so the file type could stay the same, but no worries, I'm not too fussed!

Reply:
Ah yes, I did see that, but would suggest we use the helper for consistency, if we were to do this at all 👍